networking_ccloud/tools/netbox_config_gen.py

```diff
@@ -114,6 +114,23 @@ def _ensure_single_item(cls, item_set: Iterable[Any], entity: str, identifier: s
                                         f'Unexpected values for {attribute}, expected 1 but got {len(item_set)}')
         return item_set.pop()
 
+    def get_azs(self) -> List[conf.AvailabilityZone]:
+        azs = list()
+        for site in self.netbox.dcim.sites.filter(region=self.region):
+            suffix = site.slug[len(self.region):].lower()
+            number = ord(suffix) - ord('a') + 1
+            azs.append(conf.AvailabilityZone(name=site.slug, suffix=suffix, number=number))
+        return sorted(azs, key=lambda x: x.number)
+
+    def get_cloud_vrfs(self) -> List[conf.VRF]:
+        vrfs = list()
+        search_strings = ['CC-CLOUD', 'CC-MGMT']
+        nb_return = chain(*(self.netbox.ipam.vrfs.filter(q=x, tenant=self.connection_tenants) for x in search_strings))
+        for vrf in nb_return:
+            rd_suffix = int(vrf.rd[vrf.rd.find(':') + 1:])
+            vrfs.append(conf.VRF(name=vrf.name, number=rd_suffix))  # type: ignore
+        return sorted(vrfs, key=lambda x: x.number)
```

Comment:
`key=operator.attrgetter('number')` if you want (maybe also `from operator import attrgetter`)

Reply:
okay, let's import `attrgetter` then.